server/controllers/getImage.js

```diff
@@ -0,0 +1,11 @@
+const asyncHandler = require("express-async-handler");
+const { getFileStream } = require('../utils/s3')
+
+// @route GET /images/:key
+// @desc create read stream for image from S3 bucket
+// @access Private
+exports.getImage = asyncHandler(async (req, res, next) => {
+    const imageKey = req.params.key
+    const readStream = getFileStream(imageKey);
+    readStream.pipe(res);
+});
```

Comment:
It might be more appropriate to set the necessary headers (e.g. Content-Type) before beforehand and sending the response only when the s3 stream emits `close`.